packages/ceramic-http-client/src/document.ts

```diff
@@ -1,21 +1,7 @@
-import fetch from 'node-fetch'
 import CID from 'cids'
 import { EventEmitter } from 'events'
 import cloneDeep from 'lodash.clonedeep'
-
-const fetchJson = async (url: string, payload?: any): Promise<any> => {
-  let opts
-  if (payload) {
-    opts = {
-      method: 'post',
-      body: JSON.stringify(payload),
-      headers: { 'Content-Type': 'application/json' }
-    }
-  }
-  const res = await (await fetch(url, opts)).json()
-  if (res.error) throw new Error(res.error)
-  return res
-}
+import { fetchJson } from "./utils"
```

Comment:
Can we try not to mix the use of `'` and `"` :)

Reply:
yep 👍